api/app/schemas/morecast_v2.py

```diff
@@ -0,0 +1,53 @@
+""" This module contains pydantic models for Morecast v2"""
+
+from enum import Enum
+from pydantic import BaseModel
+
+
+class ModelChoice(str, Enum):
+    """ Enumerator for all valid forecasted value types """
+    GDPS = 'GDPS'
+    GFS = 'GFS'
+    HRDPS = 'HRDPS'
+    RDPS = 'RDPS'
+    MANUAL = 'MANUAL'
+
+
+class ForecastedTemperature(BaseModel):
+    """ Forecaster chosen temperature """
+    temp: float
+    choice: ModelChoice
+
+
+class ForecastedRH(BaseModel):
+    """ Forecaster chosen rh """
+    rh: float
+    choice: ModelChoice
+
+
+class ForecastedPrecip(BaseModel):
+    """ Forecaster chosen 24-hour precipitation mm """
+    precip: float
+    choice: ModelChoice
+
+
+class ForecastedWindSpeed(BaseModel):
+    """ Forecaster chosen wind speed """
+    wind_speed: float
+    choice: ModelChoice
+
+
+class ForecastedWindDirection(BaseModel):
+    """ Forecaster chosen wind direction """
+    wind_direction: float
+    choice: ModelChoice
+
```

Comment:
Grass Cure maaaaaayyyy be required here too?

Reply:
Let's add this when we're clear who/what workflow adjusts this.